__tests__/app.test.js

```diff
@@ -113,3 +113,36 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/:article_id/comments", () => {
+  test("Should return 200 and relevant array of comments when given valid id", () => {
+    return request(app)
+      .get("/api/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((commentObj) => {
+          expect(commentObj).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
+          });
+        });
+      });
+  });
+
+  test("Should return 404 if given a valid format id that doesn't exist in comments database", () => {
+    return request(app).get("/api/99/comments").expect(404);
+  });
+
+  test("Should return 400 and send back error message if given invalid id format", () => {
```

Comment:
I'd also like to see a test that checks that the comments are sorted by created_at in descending order